docs/20260104_zoom-threshold-analysis.md

```
@@ -0,0 +1,137 @@
+| 5 | 3,200% | 9.77億円 | 数千 |
+| **6** | **6,400%** | **2.44億円** | **3,888件** |
+| 7 | 12,800% | 0.61億円 | 〜6,000件 |
+| 8 | 25,600% | 0.15億円 | 〜12,000件 |
+| 9 | 51,200% | 0.04億円 | 〜20,000件 |
+| 10 | 102,400% | 0.01億円 | 〜23,000件 |
+| 11 | 204,800% | 24万円 | ほぼ全件 |
+| **12** | **409,600%** | **6万円** | **全件（30円以上）** |
+
+### 全ノード展開に必要なズーム
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)